<launch>
	<!-- The mp3 recorder -->
    <node name="audio_recorder" type="audio_recorder" pkg="rtabmap_audio">
    	<!--
    	/media/Macintosh HD/Users/MatLab/Music/2012
    	/home/mathieu/inf_audio_saved.mp3
    	-->
    	<param name="path" value="/home/mathieu/inf_audio_saved.mp3" type="string"/>
    </node>
    
    <!-- The overlappers -->
    <node name="audio_overlap0" type="audio_overlap" pkg="rtabmap_audio">
    	<param name="ratio" value="0" type="double"/>
    	<remap from="audioOverlappedFrameFreqSqrdMagn" to="freqOverlap0" />
    </node>
    <node name="audio_overlap50" type="audio_overlap" pkg="rtabmap_audio">
    	<param name="ratio" value="0.5" type="double"/>
    	<remap from="audioOverlappedFrameFreqSqrdMagn" to="freqOverlap50" />
    </node>
    <node name="audio_overlap100" type="audio_overlap" pkg="rtabmap_audio">
    	<param name="ratio" value="1" type="double"/>
    	<remap from="audioOverlappedFrameFreqSqrdMagn" to="freqOverlap100" />
    </node>
    
    <!-- The players (only the first play the music) -->
    <node name="audio_player" type="audio_player" pkg="rtabmap_audio">
    	<param name="gui_used" value="false" type="bool"/>
    </node>
    <node name="audio_player0" type="audio_player" pkg="rtabmap_audio">
    	<remap from="audioFrame" to="dummy" />
    	<remap from="audioFrameFreqSqrdMagn" to="freqOverlap0" />
    </node>
    <node name="audio_player50" type="audio_player" pkg="rtabmap_audio">
    	<remap from="audioFrame" to="dummy" />
    	<remap from="audioFrameFreqSqrdMagn" to="freqOverlap50" />
    </node>
    <node name="audio_player100" type="audio_player" pkg="rtabmap_audio">
    	<remap from="audioFrame" to="dummy" />
    	<remap from="audioFrameFreqSqrdMagn" to="freqOverlap100" />
    </node>
    
</launch>